src/compiler/diagnosticMessages.json

```diff
@@ -1127,6 +1127,14 @@
         "category": "Error",
         "code": 1378
     },
+    "An import alias cannot reference a declaration that was exported using 'export type'.": {
+        "category": "Error",
+        "code": 1379
+    },
+    "An import alias cannot reference a declaration that was imported using 'import type'.": {
+        "category": "Error",
+        "code": 1380
+    },

```

Comment:
@DanielRosenwasser added you to look at the new error messages

Reply:
Looks good! Maybe call it an `import =` alias to be clearer, but I think this is fine.

Reply:
I think it’s weird to have non-word tokens in error messages without quotes, but I also think it’s weird to say `'import ='` because that’s not an actual excerpt of syntax (it’s missing the identifier in the middle), so I wasn’t sure how to format that 🤔

Reply:
That was my thought as well 👍